src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -69,10 +69,14 @@ var _notifier = {
   _notify: function( elem ) {
     var html = _notifier._generateHTML();
     $( html )[_notifier.settings.insertLocation]( elem )
-      .slideDown( _notifier.settings.speed, function() {
-        _notifier.existing = this;
-        if ( _notifier.settings.onRender ) {
-          _notifier.settings.onRender.call( this );
+      .slideDown( {
```

Comment:
Ah yeah, another spot where it would probably be worth defining it to reduce the code a bit.